lib/saluki-components/src/sources/checks/check_metric.rs

```diff
@@ -0,0 +1,113 @@
+use std::time::Duration;
+
+use saluki_context::{
+    tags::{Tag, TagSet},
+    Context,
+};
+use saluki_core::data_model::event::metric::{Metric, MetricMetadata, MetricValues};
+use saluki_core::data_model::event::Event;
+use tracing::warn;
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum MetricType {
+    Gauge = 0,
```

Comment:
No need for the common Go pattern of setting the value of the first variant... since it will already be zero if you cast the variant to `i32` (e.g., `MetricType::Gauge as i32`).

Only relevant if you want the offset to be something other than 0.

Reply:
Noted! Thanks for the info 😄

Reply:
Done